dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -265,41 +305,49 @@ const DropdownLink = ({ link, index }: DropdownLinkProps) => {
 			});
 
 			submitComponentEvent({
-				component: {
-					componentType: NOTIFICATION_COMPONENT_TYPE,
-					id: link.id,
-					labels: link.notifications.map((n) => n.ophanLabel),
-				},
+				component: ophanComponent,
 				action: 'VIEW',
 			});
 		}
-	}, [hasBeenSeen, link.notifications, hasSentViewEvent, link.id]);
+	}, [
+		hasBeenSeen,
+		ophanComponent,
```

Comment:
Would be worth checking but I _think_ `ophanComponent` will be considered "different" on every render by `useEffect` as it's a fresh new object each time, even if it contains the same data. Could be a case for [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo) if this is a problem?

Reply:
Oh in this case it's not a problem I don't think because `hasSentViewEvent` ensures we'll only ever send the event once. Does that sound right?

Reply:
that makes sense, but I feel safer with useMemo! Good spot